favorite-sites.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>My favorite websites</title>
+	<meta charset="utf-8">
+	<meta name="Description" content=This page show my favorite websites">
+	<meta name="keywords" content="best website, best java website, , best web development website">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+</head>
+<body>
+	<header>
+		<h1>Nazish Ahmad</h1>
+	    <p>Hi, I'm Nazish. A computer science engineering student who love to solve problem.</p>
+	</header>
+	<main>	
+	    <h2>My Favorite Websites.</h2>	
+	    <ul>

```

Comment:
You have so much information in each `<li>` that i don't think `<ul>` and `<li>` is what you should use. 

I think this is a case of `<section>` and `<article>`. Change out the `<ul />` for `<section>` and the `<li>` for `<article>`

Reply:
It was looking good by using <ul> and <li>, that's why I used.

Now I have changed it by <section> and <article>.